internal/local/command_kafka_broker_update.go

```diff
@@ -0,0 +1,104 @@
+package local
+
+import (
+	"context"
+
+	"github.com/antihax/optional"
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/kafka-rest-sdk-go/kafkarestv3"
+
+	"github.com/confluentinc/cli/v3/pkg/broker"
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafkarest"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/properties"
+)
+
+func (c *Command) newKafkaBrokerUpdateCommand() *cobra.Command {
```

Comment:
These broker commands seem completely identical to the existing broker commands. Do you think we could call the main function of those commands directly?

Reply:
how do i do this? kafka broker commands are of type `*pcmd.AuthenticatedCLICommand` 🤔

Reply:
Sounds like we'll have to create the client outside of the function, and pass it in to a the shared function?